data/dev_setup.py

```diff
@@ -57,13 +57,14 @@
                     help='Creates ArangoDB indexes for a given adapter')
 
 args = parser.parse_args()
-
 dry_run = args.dry_run
 create_indexes = args.create_indexes
 adapters = args.adapter or ADAPTERS.keys()
 
 if not dry_run:
     ArangoDB().setup_dev()
+    ADAPTERS['gencode_genes'] = GencodeGene(
+        filepath='./samples/gencode_sample.gtf', gene_alias_file_path='./samples/Homo_sapiens.gene_info.gz', dry_run=False)
 
```

Comment:
this looks misplaced

Reply:
dry_run is set to True in GencodeGene adapter by default. I need to pass the dry_run in GencodeGene adapter if dry_run is False.

Reply:
oh, I see! There are other adapters in the same situation. The dev_setup script today doesn't support dry_run=True for adapters that skip biocypher. Adding this support should be a separate ticket I think. I can work on that. We can leave dry_run default to False in the adapter for now.

Reply:
Sure